### PR TITLE
Fix issues valid golem messages

### LINKED ISSUE #36: Force report computed task use case: Missing test for AckReportComputedTask response
There is an piece of code in `if` in `receive` function which is not currently tested. There should be a test which will cover it.
```
if last_undelivered_message_status.message.type == "ForceReportComputedTask":
    if decoded_message_data.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME < current_time:
    ...
```

### PR DESCRIPTION
**Pull request by kbeker**
Resolved #36, #37, #38, #39, #43 issues. Ready for review.

### CODE REVIEW

File: concent_api/core/tests/test_core_views.py [hunk 1]
```diff
@@ -424,3 +470,67 @@ def test_view_receive_out_of_band_return_http_204_if_no_messages_in_database(sel
 
         self.assertEqual(response.status_code, 204)
         self.assertEqual(response.content.decode(), '')
+
+    def test_two_receive_out_of_band_in_row(self):
```
Comment (rwrzesien) [inline concent_api/core/tests/test_core_views.py]: I am not sure if this test expectations are correct. It means that each time somebody (for now without key checking) hits `\receive_out_of_band` he will get VerdictReportComputedTask. @kbeker @cameel is that expected behavior ?

File: concent_api/core/views.py [hunk 2]
```diff
@@ -261,123 +243,96 @@ def receive_out_of_band(request, _message):
...
+            raw_last_message_verdict,
+            None,
+            check_time = False,
+        )
+
+        last_undelivered_receive_out_of_band_status.delivered = True
+        last_undelivered_receive_out_of_band_status.full_clean()
+        last_undelivered_receive_out_of_band_status.save()
 
-    return None
```
Comment (rwrzesien) [inline concent_api/core/views.py]: I think it should still `return None` at the very end of this function.

File: concent_api/core/views.py [hunk 3]
```diff
@@ -261,123 +243,96 @@ def receive_out_of_band(request, _message):
...
             )
-            store_receive_out_of_band(golem_message, message_timestamp)
+            store_receive_out_of_band(
+                golem_message,
+                message_timestamp
+            )
             message_verdict.sig = None
             return message_verdict
+        return None
+    else:
```
Comment (rwrzesien) [inline concent_api/core/views.py]: According to my previous comment about test I think this `else:` part can should be removed too.

File: concent_api/core/tests/test_core_views.py (hunk 1 above)
Comment (cameel) [reply concent_api/core/tests/test_core_views.py]: The test is wrong and needs to be changed. `VerdictReportComputedTask` should be returned only once.

By the way the name of the test function does not follow our convention. The pattern is `def test_<what you're testing>_should_<what you expect it to do>():`

**Pull request closed without merging**